### PR TITLE
Good practices to track events in Firebase

### PR DESCRIPTION
**Pull request by tpucci**

### CODE REVIEW

**Comment by bam-bot-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/tpucci/debug-firebase-analytics/react-native/firebase/debug-events.mo.md">debug-events.mo.md</a>**: Seems you do not need a *Troubleshoot* part
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: react-native/firebase/debug-events.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+_Example:_
+
+![Trello tickets](./assets/example-tickets.png)
+
+## Steps
+
+### Set up your device in debug mode (10min)
+
+Choose to start with either Android or iOS. Once you've finished with an OS, swith to the other.
```
Comment (Minishlink) [inline react-native/firebase/debug-events.md]: switch

File: react-native/firebase/debug-events.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+> **Check**:
+>
+> 1. In Firebase, in the left menu, go in **Analytics**, **Debug View**.
+> 2. Select your application in the top menu.
+> 3. Your debug devices count should print: **APPAREIL DE DÉBOGAGE: 0**
+
+#### Android
+
+1. Run your simulator or connect your device.
```
Comment (Minishlink) [inline react-native/firebase/debug-events.md]: Reloading the simulator doesn't suffice?

File: react-native/firebase/debug-events.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+## Owner: [Thomas Pucci](https://github.com/tpucci)
+
+## Prerequisites
+
+* [React Native Firebase](https://github.com/invertase/react-native-firebase) is installed on your project
+* You want to track one event
+
+## Scrum
+
+* You should have one ticket to send the event to Firebase which your PO will validate with a screenshot.
```
Comment (Minishlink) [inline react-native/firebase/debug-events.md]: Maybe "Validating events tracking with Firebase should be done with two tickets:
1. ...
2. ...
"

**Review by Minishlink — APPROVED** [review]

File: react-native/firebase/debug-events.md (hunk 3 above)
Comment (tpucci) [reply react-native/firebase/debug-events.md]: 👍

File: react-native/firebase/debug-events.md (hunk 1 above)
Comment (tpucci) [reply react-native/firebase/debug-events.md]: 👍

File: react-native/firebase/debug-events.md (hunk 2 above)
Comment (tpucci) [reply react-native/firebase/debug-events.md]: :+1:

**Review by tychota — CHANGES_REQUESTED** [review]
Please use a `.mo.md` file as requested by danger

**Comment by tychota** [comment]
Ajouute le a summary.md

**Review by tychota — APPROVED** [review]

**Pull request merged**